account_analytic_parent/views/account_analytic_account_view.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_account_analytic_account_form" model="ir.ui.view">
+        <field name="name">account.analytic.account.form</field>
+        <field name="model">account.analytic.account</field>
+        <field name="type">form</field>
+        <field name="inherit_id" ref="analytic.view_account_analytic_account_form"/>
+        <field name="arch" type="xml">
+            <field name="tag_ids" position="after">
+                <field name="parent_id" string="Parent Analytic Account"/>
+            </field>
+        </field>
+    </record>
+
+    <record id="view_account_analytic_account_report" model="ir.ui.view">
```

Comment:
As `field_parent` is no longer supported, get rid of this view, unless someone can think of a way to replicate this in another way.